### ISSUE TITLE
Uncaught TypeError: Error processing argument at index 0, conversion failure from null

### ISSUE THREAD

**Issue by Tikam02** [atom/deprecation-cop#76]
### Prerequisites

* [ ] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: http://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

[Description of the issue]

### Steps to Reproduce

1. [First Step]
2. [Second Step]
3. [and so on...]

**Expected behavior:** [What you expect to happen]

**Actual behavior:** [What actually happens]

**Reproduces how often:** [What percentage of the time does it reproduce?]

### Versions

You can get this information from copy and pasting the output of `atom --version` and `apm --version` from the command line. Also, please include the OS and what version of the OS you're running.

### Additional Information

Any additional information, configuration or data that might be necessary to reproduce the issue.

**Comment by Ben3eeE** [comment]
Thanks for taking the time to contribute!

We noticed that this is a duplicate of https://github.com/atom/deprecation-cop/issues/68. You may want to subscribe there for updates.

Because we treat our issues list as [the Atom team's backlog](https://en.wikipedia.org/wiki/Scrum_(software_development)#Product_backlog), we close duplicates to focus our work and not have to touch the same chunk of code for the same reason multiple times. This is also why we may mark something as duplicate that isn't an exact duplicate but is closely related.

For information on how to use GitHub's search feature to find out if something is a duplicate before filing, see [the How Can I Contribute? section of the Atom CONTRIBUTING guide](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#how-can-i-contribute).

**Comment by V1rtus** [comment]
Starting from Atom v1.13.0, the contents of atom-text-editor elements are no longer encapsulated within a shadow DOM boundary. This means you should stop using :host and ::shadow pseudo-selectors, and prepend all your syntax selectors with syntax--. To prevent breakage with existing style sheets, Atom will automatically upgrade the following selectors:

.atom-text-editor, :host => .atom-text-editor,atom-text-editor
.atom-text-editor .invisible-character, :host .invisible-character => .atom-text-editor .invisible-character,atom-text-editor .invisible-character
.atom-text-editor .indent-guide, :host .indent-guide => .atom-text-editor .indent-guide,atom-text-editor .indent-guide
.atom-text-editor .wrap-guide, :host .wrap-guide => .atom-text-editor .wrap-guide,atom-text-editor .wrap-guide
.atom-text-editor .gutter, :host .gutter => .atom-text-editor .gutter,atom-text-editor .gutter
.atom-text-editor .gutter .cursor-line, :host .gutter .cursor-line => .atom-text-editor .gutter .cursor-line,atom-text-editor .gutter .cursor-line
.atom-text-editor .line-number.cursor-line-no-selection, :host .line-number.cursor-line-no-selection => .atom-text-editor .line-number.cursor-line-no-selection,atom-text-editor .line-number.cursor-line-no-selection
.atom-text-editor .invisible, :host .invisible => .atom-text-editor .invisible,atom-text-editor .invisible
.atom-text-editor .cursor, :host .cursor => .atom-text-editor .cursor,atom-text-editor .cursor
.atom-text-editor .selection .region, :host .selection .region => .atom-text-editor .selection .region,atom-text-editor .selection .region
.atom-text-editor .search-results .marker .region, :host .search-results .marker .region => .atom-text-editor .search-results .syntax--marker .region,atom-text-editor .search-results .syntax--marker .region
.atom-text-editor .search-results .marker.current-result .region, :host .search-results .marker.current-result .region => .atom-text-editor .search-results .syntax--marker.current-result .region,atom-text-editor .search-results .syntax--marker.current-result .region
.atom-text-editor .gfm .markup.heading, :host .gfm .markup.heading => .atom-text-editor .syntax--gfm .syntax--markup.syntax--heading,atom-text-editor .syntax--gfm .syntax--markup.syntax--heading
.atom-text-editor .gfm .markup.underline, :host .gfm .markup.underline => .atom-text-editor .syntax--gfm .syntax--markup.syntax--underline,atom-text-editor .syntax--gfm .syntax--markup.syntax--underline
.atom-text-editor .gfm .bold, :host .gfm .bold => .atom-text-editor .syntax--gfm .syntax--bold,atom-text-editor .syntax--gfm .syntax--bold
.atom-text-editor .gfm .italic, :host .gfm .italic => .atom-text-editor .syntax--gfm .syntax--italic,atom-text-editor .syntax--gfm .syntax--italic
.atom-text-editor .gfm .raw, :host .gfm .raw => .atom-text-editor .syntax--gfm .syntax--raw,atom-text-editor .syntax--gfm .syntax--raw
.atom-text-editor .gfm .variable.list, :host .gfm .variable.list => .atom-text-editor .syntax--gfm .syntax--variable.syntax--list,atom-text-editor .syntax--gfm .syntax--variable.syntax--list
.atom-text-editor .gfm .link, :host .gfm .link => .atom-text-editor .syntax--gfm .syntax--link,atom-text-editor .syntax--gfm .syntax--link
.atom-text-editor .gfm .link .entity, :host .gfm .link .entity => .atom-text-editor .syntax--gfm .syntax--link .syntax--entity,atom-text-editor .syntax--gfm .syntax--link .syntax--entity
.comment => .syntax--comment
.string => .syntax--string
.constant.numeric => .syntax--constant.syntax--numeric
.constant.language => .syntax--constant.syntax--language
.constant.character, .constant.other => .syntax--constant.syntax--character, .syntax--constant.syntax--other
.keyword => .syntax--keyword
.storage => .syntax--storage
.storage.type => .syntax--storage.syntax--type
.entity.name.class => .syntax--entity.syntax--name.syntax--class
.entity.other.inherited-class => .syntax--entity.syntax--other.syntax--inherited-class
.entity.name.function => .syntax--entity.syntax--name.syntax--function
.entity.name.instance => .syntax--entity.syntax--name.syntax--instance
.variable.parameter => .syntax--variable.syntax--parameter
.entity.name.tag => .syntax--entity.syntax--name.syntax--tag
.entity.other.attribute-name => .syntax--entity.syntax--other.syntax--attribute-name
.support.function => .syntax--support.syntax--function
.support.constant => .syntax--support.syntax--constant
.support.type, .support.class => .syntax--support.syntax--type, .syntax--support.syntax--class
.invalid => .syntax--invalid
.invalid.deprecated => .syntax--invalid.syntax--deprecated
.class.jade => .syntax--class.syntax--jade
Automatic translation of selectors will be removed in a few release cycles to minimize startup time. Please, make sure to upgrade the above selectors as soon as possible.

**Comment by lewismoten** [comment]
Although this issue, and the issue it duplicates are closed, this problem still exists as of 1.13.1
```

The contents of atom-text-editor elements are no longer encapsulated within a shadow DOM boundary. Please, stop using shadowRoot and access the editor contents directly instead.
Report Issue
atom-text-editor.Object.defineProperty.get - C:\Users\lewismoten\AppData\Local\atom\app-1.13.1\resources\app.asar\src\text-editor-element.js:68:18
<unknown> - C:\Users\lewismoten\.atom\packages\css-color-underline\lib\main.coffee:14:29
Function.module.exports.Emitter.simpleDispatch - C:\Users\lewismoten\AppData\Local\atom\app-1.13.1\resources\app.asar\node_modules\event-kit\lib\emitter.js:25:14
Emitter.emit - C:\Users\lewismoten\AppData\Local\atom\app-1.13.1\resources\app.asar\node_modules\event-kit\lib\emitter.js:129:28
<unknown> - C:\Users\lewismoten\AppData\Local\atom\app-1.13.1\resources\app.asar\src\text-editor.js:541:32
Function.module.exports.Emitter.simpleDispatch - C:\Users\lewismoten\AppData\Local\atom\app-1.13.1\resources\app.asar\node_modules\event-kit\lib\emitter.js:25:14
```
